assets/src/edit-story/components/canvas/test/multiSelectionMovable.js

```diff
@@ -122,17 +122,19 @@ describe('multiSelectionMovable', () => {
     'should rotate %p',
     (_, { rotateTo, expectedRotationAngle }) => {
       arrange();
+
       performRotatation(rotateTo);
```

Comment:
Can you please rename this while you're here? _"rota**ta**tion"_?

Reply:
Will do, it sounds nice though.